climakitae/data/variable_catalog.csv

```diff
@@ -1,49 +1,52 @@
 variable_id,long_name,timescale,unit,display_name,extended_description,colormap,show
+t2,temp at 2 m,hourly,K,Air Temperature at 2m,Temperature of the air 2m above Earth's surface. This is the measure of air temperature used for most modeling applications.,ae_orange,TRUE
+precip_tot_derived,Precipitation (total),hourly,mm,Precipitation (total),Total precipitation. Computed by summing total gridscale precipitation and total cumulus precipitation.,ae_blue,TRUE
+rh_derived,Relative Humidity,hourly,[0 to 100],Relative Humidity,The percentage of water vapor (i.e. moisture) relative to the amount of water vapor needed to saturate the air at the current air temperature and pressure.,ae_blue,TRUE
+wind_mag_derived,Wind Magnitude,hourly,m s-1,Wind Magnitude at 10m,Wind speed at 10 meters above the Earth's surface. Computed by taking the vector magnitude of the west-east (u) component of wind and the north-south (v) component of wind.,ae_orange,TRUE

```

Comment:
For wind magnitude, which we derive, everything looks fine when the data is called in at hourly timescales except for the variable_id which is set to u10 (which we define as for the west-east component of the wind). 

This is fine and somewhat confusing difference but "u10" is technically okay as long as it's a capital U, so "U10" for wind magnitude. But lower-case "u10" is specifically for E-W wind. But since I can see "wind_mag_derived" that would be better to include so things don't get confusing.

Reply:
To clarify: my preference for the wind magnitude variable is to have the variable_id be set to "wind_mag_derived"